packages/tools/src/tools/base/ContourSegmentationBaseTool.ts

```diff
@@ -79,17 +75,15 @@ abstract class ContourSegmentationBaseTool extends ContourBaseTool {
       return contourAnnotation;
     }
 
-    const activeSeg = getActiveSegmentationRepresentation(viewport.id);
+    const activeSeg = getActiveSegmentation(viewport.id);
```

Comment:
Here is where it would be nice to just include the type, and then if !activeSeg, it simply will throw the single error.